### ISSUE TITLE
Mod 4 - Submit a new expenditure - expenditures/new

### ISSUE THREAD

**Issue by danmelton** [hackoregon/openelections#575]
![civic__elections--ux](https://user-images.githubusercontent.com/99731/62509523-66559780-b7c0-11e9-9ff5-7709bd227646.png)

- [x] Add a new route for /expenses/new
- [x] Title the screen 'New Expenditure', do not show an id, we dont have one yet
- [x] Ensure that the type dropdowns effect the subtype dropdowns
- [x] Ensure that all the required fields are present (look at API/models/entities/expenditures) file if unsure which ones are required for submission
- [x] The Save as draft button should be greyed out until all required fields are present
- [x] Add * to all required fields
- [x] Saving the draft should take you to the expenditures/:id page

User fills out all relative form fields
Basics
Amount - the amount of the Expense
Date of Expenditure - the date the expense was Paid or Incurred 
Type of Expenditure - Expenditure, Other, Other Disbursement
Subtype of Expenditure - If Expenditure Type is “Expenditure,” drop down says: Accounts Payable, Cash Expenditure, Personal Expenditure for Reimbursement.
If Expenditure Type is “Other.” drop down says: Accounts Payable Rescinded, Cash Balance Adjustment (maybe)
If Expenditure Type is “Other Disbursement,” drop down says: Miscellaneous Other
Payment Method - drop down provides options of Check, Credit Card, Debit Card, Electronic Check, Electronic Funds Transfer
Check number - If the expense was paid for with a check, the check number goes here 
Payee Information
Payee Type - select option from drop down
Payee Name - the name of the recipient of payment
Address information of recipient - fill out all  address information regarding the recipient. 

Other Details
Purpose of Expenditure - Why was this Expense encountered? 
Notes - if it will be helpful to have information regarding this transaction in the future feel free to add that information here.

### Out of Scope
- Labels
- activity (there isnt any)
- submit expenditure button

**Comment by punkael** [comment]
Blocked by #724 
After #724 is fixed remove default value for expenditure purpose.

**Comment by kdv24** [comment]
@punkael I'm not finding a default value set for expenditure purpose, but before you merge in https://github.com/hackoregon/openelections/pull/721, can you double check that I'm not misunderstanding? Thanks!

Then I think #575 can be closed again?
